.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
Why has this changed?

Reply:
Trusty is really old

Reply:
Ended support in 2016, so using a newer version doesn't hurt- and we get newer built in libs with it